### ISSUE TITLE
Quest kill objectives sometimes not awarded if killed by totems

### ISSUE THREAD

**Issue by WhitePrideWorldWide** [ccshiro/cc-buglist#2562]
These 2 quests: Wood mice on Infected Protectors when killed by an AOE totem, and Murkblood Scavenger for Once Were Warriors with Searing Totem.  Other quests I tried after I noticed it seemed to work.

**Comment by WhitePrideWorldWide** [comment]
Update on this, I think it only happens if totems do 100% of the damage to a mob, in which case you get no credit and no loot.  Even 1 tick of lightning shield damage cancels this.

**Comment by KimmoKM** [comment]
That's how it worked with pets so it's possible it might be the case for totems as well. I'd imagine it'd be difficult to prove either way.

**Comment by Ainulatac** [comment]
I agree with KimmoKM 100%, must be that, so allegedly works as intended (Blizzlike).

**Comment by WhitePrideWorldWide** [comment]
So you're saying if you do a low level instance: you pull the whole thing and use your only aoe, which is totems.  Then you get no loot and your buddy gets no xp?  Also the wood mice spawn neutral 5+ with 1 hp and I don't remember sitting there auto attacking them, so I don't believe it's blizzlike.

**Comment by KimmoKM** [comment]
Makes sense. Yeah, now that you mention those examples, totems probably were handled differently from pets.
